colibre/registration.py

```diff
@@ -94,7 +94,38 @@ def register_spesific_star_formation_rates(self, catalogue, aperture_sizes):
             (stellar_mass > unyt.unyt_quantity(1e10, units="Msun")).astype(float),
             units="dimensionless",
         )
-        is_active.name = "Stellar mass larger than 10^10 Msun"
+        is_bigger_than_1e10.name = "Stellar mass larger than 10^10 Msun"
+
+        # Mask for galaxies above 10^10 Msun
+        is_bigger_than_1e10_active = unyt.unyt_array(
+            (
+                (stellar_mass > unyt.unyt_quantity(1e10, units="Msun"))
```

Comment:
Some recent experiences have taught us that you cannot rely on `unyt` doing the right thing when comparing `unyt_array`s; it tends to just completely ignore units. I therefore propose the following:
```
stellar_mass > (1.e10*unyt.Msun).to(stellar_mass.units)
...
ssfr > (1.01*marginal_ssfr).to(ssfr.units)
```